tensorboard/webapp/feature_flag/effects/feature_flag_effects.ts

```diff
@@ -16,17 +16,27 @@ limitations under the License.
 import {Injectable} from '@angular/core';
 import {Actions, createEffect, ofType} from '@ngrx/effects';
 import {Action, createAction, Store} from '@ngrx/store';
-import {combineLatestWith, map} from 'rxjs/operators';
+import {combineLatestWith, map, tap, withLatestFrom} from 'rxjs/operators';
+import '../../tb_polymer_interop_types';
 import {TBFeatureFlagDataSource} from '../../webapp_data_source/tb_feature_flag_data_source_types';
 import {partialFeatureFlagsLoaded} from '../actions/feature_flag_actions';
 import {ForceSvgDataSource} from '../force_svg_data_source';
-import {getIsAutoDarkModeAllowed} from '../store/feature_flag_selectors';
+import {
+  getFeatureFlags,
+  getIsAutoDarkModeAllowed,
+} from '../store/feature_flag_selectors';
 import {State} from '../store/feature_flag_types';
 
 const effectsInitialized = createAction('[FEATURE FLAG] Effects Init');
 
 @Injectable()
 export class FeatureFlagEffects {
+  // Ngrx assumes all Effect classes have properties that inherit from the base
+  // JS Object. `tf_feature_flags` does not, so we wrap it.
+  private readonly tfFeatureFlags = {
```

Comment:
does this mean that this object is owned by the FeatureFlagEffects class?   

Does that |initializeFeatureFlags| function get called when creating this element? Is that what the self assigned variable does?